File: src/main/java/sigmaone/industrialism/energy/IBlockEnergyHandler.java
package sigmaone.industrialism.energy;

import net.minecraft.util.math.BlockPos;

public interface IBlockEnergyHandler {
    /*
     * Transfer energy to a block at position pos
     *
     * @param pos BlockPos of the recipient
     */
    void sendEnergy(BlockPos pos, float amount);
}

File: src/main/kotlin/sigmaone/industrialism/energy/IBlockEnergyHandler.kt
package sigmaone.industrialism.energy

import net.minecraft.util.math.BlockPos

interface IBlockEnergyHandler {
    /*
     * Transfer energy to a block at position pos
     *
     * @param pos BlockPos of the recipient
     */
    fun sendEnergy(pos: BlockPos?, amount: Float)
}